packages/client/components/BottomControlBarReady.tsx

```diff
@@ -132,29 +171,48 @@ const BottomControlBarReady = (props: Props) => {
     if (phaseType === 'reflect') {
       return reflectionGroups?.length === 0 ?? true
     }
+
+    if (phaseType === 'vote') {
+      return !hasVotes
+    }
+
     return false
   }
   const disabled = getDisabled()
+  const onMouseOver = () => {
+    if (phaseType === 'vote' && disabled) {
+      openErrorTooltip()
+    }
+  }
+
   return (
     <>
-      <BottomNavControl
-        dataCy={`next-phase`}
+      <BottomNavContorlWrapper
         disabled={disabled}
-        confirming={!!cancelConfirm}
-        onClick={cancelConfirm || onClick}
-        status={status}
-        onTransitionEnd={onTransitionEnd}
-        onKeyDown={onKeyDown}
-        ref={ref}
+        onMouseOver={onMouseOver}
+        onMouseLeave={closeErrorTooltip}
+        ref={errorTooltipRef}
       >
-        <BottomControlBarProgress isNext={isNext} progress={progress} />
-        <BottomNavIconLabel label={label} ref={originRef}>
-          <CheckIcon isViewerReady={isViewerReady} isNext={isNext} progress={progress}>
-            {icon}
-          </CheckIcon>
-        </BottomNavIconLabel>
-      </BottomNavControl>
+        <BottomNavControl
+          dataCy={`next-phase`}
+          disabled={disabled}
+          confirming={!!cancelConfirm}
+          onClick={cancelConfirm || onClick}
+          status={status}
+          onTransitionEnd={onTransitionEnd}
+          onKeyDown={onKeyDown}
+          ref={ref}
+        >
+          <BottomControlBarProgress isNext={isNext} progress={progress} />
+          <BottomNavIconLabel label={label} ref={originRef}>
+            <CheckIcon isViewerReady={isViewerReady} isNext={isNext} progress={progress}>
+              {icon}
+            </CheckIcon>
+          </BottomNavIconLabel>
+        </BottomNavControl>
+      </BottomNavContorlWrapper>
       {tooltipPortal(`Tap 'Next' again to Confirm`)}
+      {errorTooltipPortal(`At least one vote required`)}
```

Comment:
I feel this message not quite user friendly but I don't know what would be a better version... tagging @avivapinchas

Reply:
How about: 
> Need to vote at least once to advance

I'm not a huge fan of this one either, but not sure what would be better. 

This is perhaps a bigger question, but: why do they need to vote at least once? It seems like an un-needed gate.

I find when copy is hard to write, it's often because the thing we're trying to explain doesn't make sense on some level.

Reply:
Will modify the PR to just allow users to advance to the next phase without voting